cypress/integration/event-signup-flow.spec.js

```diff
@@ -646,3 +646,49 @@ describe(`Feature - 404 Not Found unknown_route : ${new Date()}`, () => {
 		cy.verify404ErrorMessage();
 	});
 });
+
+describe("Verify page load " + new Date(), () => {
```

Comment:
Can we add a test that interacts with the month picker drop down as well? it defaults to all months by default, so if we can maybe just select the first month and ensure it loads ok?

Reply:
Implemented